Makefile

```diff
@@ -289,6 +294,14 @@ ifndef VERSION
 	$(error VERSION must be set)
 endif
 
+.PHONY: check-previous-release-tag
+check-previous-release-tag:
+	@if [ -z "${PREVIOUS_VERSION}" ]; then echo "PREVIOUS_VERSION is not set"; exit 1; fi
+
+.PHONY: check-github-token
```

Comment:
Could you add a comment here, whose GitHub token is required? This will give better understanding to the person reading code.

Reply:
For automated releases, a GITHUB_TOKEN is required (who has the repo access). Please correct me if I am wrong. 
I will add a comment and update the PR. 
Also, should I modify the `docs/releasing.md` and add about GitHub Release notes generation steps?